docs/compiler/cli/testing.md

```diff
@@ -0,0 +1,47 @@
+# Testing of your application
+
+Testing of your qooxdoo app can be done by the qooxdoo cli command `qx test`.
+For this purpose you need to install one of the provide test plugins:
+  - qooxdoo/qxl.testtapper: Runs units tests based on `qx.dev.unit.TestCase`
+
+## Preparing your application
+
+  - install testapper with `qx package install qooxdoo/qxl.testtapper`
+  - because testtapper is only needed for development remove the testtapper from `Manifest.json` in the

```

Comment:
If you do `qx package install qooxdoo/qxl.testtapper --save=0` , no need to remove entry from manifest as it will not be saved there.

Reply:
Thanks for the hint!